src/components/CircularCountDownTimer.vue

```diff
@@ -343,22 +361,17 @@
         },
         mounted: function () {
             this.$nextTick(() => {
-                this.baseTime = Date.now();
                 this.isMounted = true;
                 if(this.value){
                     const interval = setInterval(function () {
                         if(this.paused){
                             return;
                         }
-                        const delta = Math.floor((Date.now() - this.baseTime) / 1000);
-                        this.value = this.initialValue - delta;
+                        const delta = 1
+                        this.value -= delta;
                         if(this.value === 0){
                             this.$emit('finish');
                         }
-                        if(this.value <= 0 && !this.showNegatives){
```

Comment:
Why you have deleted this lines?

Reply:
It might have been a bit silly to remove that line. My use case requires to add or subtract seconds from this.value without showing negatives. The issue I faced was that some times the subtraction leads to this.value<=0, which was stopping the timer. Since there is no method to restart from a stopped state, I deleted this block as a quickfix. This is hardly ideal though